styhead/rz sbc: rc2 fix and improvements

tiennguyenuh opened the pull request:
This PR includes:
- Update readme
- Change default tune to Cortex A55
- Enable Tk installation for python3
- Change Qt inc name
- Add Ethernet patch to prevent eth0 from being disabled before eth1

tiennguyenuh committed:
rz-sbc: update README, linux recipe and configuration

Changes include:
	- Set the default tune to cortexa55 instead of armv8a for
          optimization on the RZ/G2L's Cortex-A55 core
	- Update README to remove chromium section
	- Use rzg2l-sbc_defconfig instead of defconfig
	- Add docker.cfg to the Linux Yocto recipe when DISTRO=ubuntu-tiny
	- Enable tkinter installation for python3

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

tiennguyenuh committed:
rz-cmn-srp-2.0: change qt inc file name and add audio related-config

Changes include:
	- Rename Qt include file for clarity and consistency with its
	  Qt-specific usage
	- Add `CONFIG_SND_ALOOP` to the kernel config fragment
	- Update README to reflect the new tune changes

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

tiennguyenuh committed:
rz-cmn-srp: add Ethernet patch fix

Add Ethernet patch to prevent eth0 from being disabled before eth1.

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>

vudangRVC commented on recipes-docs/rzg2l-sbc-readme/files/README.md:
```diff
@@ -8,8 +8,8 @@ This is a VLP v3.0.5 release of the RZG2L development product for RZG2L-SBC.
 This release provides the following features:
 
  - Yocto build compatible with RZG2L SoC (VLP v3.0.5)
```
You should mention that VLP version here is intended to control the version of IPL (flash-writer, atf, uboot).

Other stacks are completely in different versions

vudangRVC replied:
It's fine then as we've already mentioned the Linux kernel version

vudangRVC commented on recipes-kernel/linux/rzg2l-sbc/dts-patches/0011-rzg2l-sbc-Prevent-disable-eth0-before-eth1.patch:
```diff
@@ -0,0 +1,110 @@
+Date: Thu, 20 Mar 2025 10:44:25 +0000
+Subject: [PATCH 11/11] rzg2l-sbc: Prevent disable eth0 before eth1
+
+This commit was created to fix the issue where eth1 could not be used
```
Please update this message by using a formal tone

tiennguyenuh committed:
rz-cmn-srp: Add default BB_PRESSURE_MAX_MEMORY setting

Changes include:
	- Add BB_PRESSURE_MAX_MEMORY with a default value of `100000` in
	  local.conf
	- Add a note to inform users that this value should be adjusted
	  based on their system resources

Signed-off-by: Tien Nguyen <tien.nguyen.uh@renesas.com>
